lib/claim_letters/claim_letter_downloader.rb

```diff
@@ -65,15 +70,17 @@ def filter_letters(document)
 
     def format_letter_data(docs)
       letters = docs.map { |d| filter_letters(d) }.compact
+      # TODO: (rare) Handle nil received_at
```

Comment:
Is this issue outstanding?

`# TODO: (rare) Handle nil received_at`

Reply:
`received_at` is autogenerated, and we haven't seen a case where it is ever `nil`. This is just a TODO to add in some extra diligence when we have a chance. My other team members are ok to get this merged as is (cc: @kjduensing )

Reply:
@holdenhinkle how do you feel about this? The chance of this happening is slim, but not none. Are y'all ok with us leaving a `TODO` in here to address later?

Reply:
Are the values for claim letters' received_at typically the time they are downloaded?

Reply:
@RachalCassity it's the time that the system we are pulling from received the file

Reply:
@jerekshoe Can you make a ticket to address this?